components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -291,8 +290,7 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
 
     this.serviceGroupsSuggestions$ = this.store.select(createSelector(serviceGroupsState,
       (serviceGroups) => serviceGroups.suggestions.values))
-        .pipe(map((serviceGroupsSuggestions:
-          ServiceGroupsSuggestions[]) => serviceGroupsSuggestions));
+        .pipe(map(serviceGroupsSuggestions => serviceGroupsSuggestions));
```

Comment:
The explicit type was actually wrong; I just removed it because the type is properly inferred.
But this is a good example showing how the newer version of TypeScript is tighter, reporting an error that our prior version did not.